### ISSUE TITLE
Browsersync doesn't compile or watch files.

### ISSUE THREAD

**Issue by myleshyson** [JeffreyWay/laravel-mix#1538]
- Laravel Mix Version: 2.1.7
- Node Version (`node -v`): 9.8.0 (also tried on 8.9.4 and 8.10)
- NPM Version (`npm -v`): 5.6.0
- Yarn Version: 1.5.1
- OS: Mac High Sierra 10.13.3

### Description:
Whenever I run `yarn run watch`, the console says `watching files` and then says `done`. It doesn't even compile my files. Here is an image with what I'm talking about

<img width="779" alt="mix" src="https://user-images.githubusercontent.com/11747187/37559645-e86032c6-29ff-11e8-8a9c-312fc8db8c9e.png">

Here is my mix file.

```js
let mix = require('laravel-mix');

mix.js('resources/assets/js/app.js', 'public/js')
   .sass('resources/assets/sass/app.scss', 'public/css')
   .setPublicPath('public')
   .browserSync({
       proxy: 'mix.test',
       files: [
        'public/**/*',
        'resources/views/**/*.*'
       ]
   });
```

Also when running `yarn run watch` for the first time, it doesn't install the browsersync dependencies on the first try. I have to run `yarn run watch` twice for the dependencies to fully download.

### Steps To Reproduce:

`laravel new mix`
`yarn install`
`yarn run watch` ...to install the browsersync dependencies, though it makes me do this twice for some reason.
`yarn run watch`

**Comment by aryehraber** [comment]
I ran into the same issue today! Didn't figure out _what_ was wrong but rolling back to a previous version of Browsersync's webpack plugin seemed to work.

In your `package.json` file, update `browser-sync-webpack-plugin` to `2.0.1` and reinstall your deps.

Or you can run:
```
yarn add -D browser-sync-webpack-plugin@2.0.1
```

Hopefully an official fix will be realised soon.

**Comment by tetshi** [comment]
+1 for this issue. Since the new tailwind implementation requires Mix 2.0 to take example of Extends (Apparently, 1.7 didn't have that), you have to update to use that, but then it breaks watching.

**Comment by JeffreyWay** [comment]
Yeah, looks like there was some breaking change in `browser-sync-webpack-plugin`. I've tagged a new release that hardcodes the version requirement to 2.0.1 for now. 

I've also fixed the issue that required two `npm run watch`'s to install both dependencies. Both tagged in 2.1.8.

**Comment by aryehraber** [comment]
@JeffreyWay Thanks for patching this up so quickly!

**Comment by malinushj** [comment]
Made a pull request to fix this issue, waiting to get merged.

**Comment by malinushj** [comment]
@JeffreyWay Official plugin fix was released: `2.2.2` https://github.com/Va1/browser-sync-webpack-plugin/releases/tag/v2.2.2
